hpOneView/resources/settings/versions.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright (2012-2017) Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+###
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from future import standard_library
+
+standard_library.install_aliases()
+
+from hpOneView.resources.resource import ResourceClient
+
+
+class Versions(object):
+    """
+    Version API client. It indicates the range of API versions supported by the appliance.
+
+    """
+    URI = '/rest/version'
+
+    def __init__(self, con):
+        self._client = ResourceClient(con, self.URI)
+
+    def get(self, id_or_uri):
+        """
+        Retrieves current version and minimum version.
+
+        Note:
+            This request does not require Auth or X-Api-Version headers
+            
+        Returns:
+            dict: version
+        """
+        version = self._client.get(id_or_uri)
```

Comment:
This could be improved in a similar fashion to the one mentioned in the [previous comment](https://github.com/HewlettPackard/python-hpOneView/pull/326/files#r147170798)